approval_polls/tests.py

```diff
@@ -161,6 +161,36 @@ def test_vote_view_counts_increase(self):
         self.assertContains(response, '21 votes')
         self.assertContains(response, '101 ballots', status_code=200)
 
+class MyPollTestsWithoutLoggedIn(TestCase):
+    def setUp(self):
+        self.client = Client()
+        user = User.objects.create_user('test','test@example.com','test')
+
+    def test_my_polls_without_logged_in(self):
+        """
+        If user is not loggin in, it return a 302 found.
```

Comment:
The description would be better written as 'If the user is not logged in then redirect to the login page'. 
Can you also check in the test if the url that is redirected to is the login page?